dvc/data_cloud.py

```diff
@@ -14,6 +14,7 @@
 from dvc.remote.hdfs import RemoteHDFS
 from dvc.remote.local import RemoteLOCAL
 from dvc.remote.http import RemoteHTTP
+from dvc.remote.config import get_remote_settings
```

Comment:
jfyi - there is a unresolved reference in the docstring for the DataCloud class below ... config is not defined